README.md

```diff
@@ -31,3 +31,28 @@
 - **Next edit the [catalog-source.yaml](config/samples/catalog-source.yaml) template to indicate your new Quay.io org image**  
 - `make release catalog-update`
 - search the Crunchy Bridge Operator in OperatorHub, click on install.
+
+## DBaaS Integration
+
+Follow the steps below to enable integration with DBaaS.
```

Comment:
This needs some clarification. The 3 steps below are not needed to integrate with DBaaS. If anything, they're needed to unit test the functionality **without** integrating with the DBaaS operator

Reply:
> This needs some clarification. The 3 steps below are not needed to integrate with DBaaS. If anything, they're needed to unit test the functionality **without** integrating with the DBaaS operator

Thanks, @bmozaffa, updated the description.